python/paddle/tests/test_model.py

```diff
@@ -718,6 +718,29 @@ def test_dygraph_export_deploy_model_about_inputs(self):
         model.save(save_dir, training=False)
         shutil.rmtree(save_dir)
 
+    def test_accumulate(self, ):
```

Comment:
better to check gradients between  one large batch size and multiple small batch sizes

Reply:
done